zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -636,7 +636,7 @@ public String create(final String path, Object datat, final List<ACL> acl, final
    * @throws IllegalArgumentException if called from anything else except the ZooKeeper event thread
    * @throws ZkException if any zookeeper exception occurs
    */
-  private String create(final String path, final Object dataObject, final List<ACL> acl,
+   public String create(final String path, final Object dataObject, final List<ACL> acl,
```

Comment:
Why change this implementation? This is interface change.

Reply:
@kaisun2000 This is not an interface change. This method is the base method for node creation. When I implemented it, I intentionally made it private not to allow direct call of `create(CreateMode.persistent, expectedSessionId)`.
I also had the question for changing it public. But here, the implementation needs to call `rawZkClient.create()` in `DedicatedZkClient` which is put in a different package. And this needs `public`. If we could put `DedicatedZkClient` in a same package with `ZkClient`, we don't need it public.

Reply:
@pkuwm Said it.